src/harness/utils.ts

```diff
@@ -86,4 +86,8 @@ namespace utils {
     export function addUTF8ByteOrderMark(text: string) {
         return getByteOrderMarkLength(text) === 0 ? "\u00EF\u00BB\u00BF" + text : text;
     }
+
+    export function isTsConfigFile(file: { path: string }): boolean {

```

Comment:
This should probably be in _vpath.ts_ with the other `is` path methods.

Reply:
Moved.